etc/scripts/kpp_to_micm.py

```diff
@@ -129,12 +135,14 @@ def micm_species_json(lines, fixed=False, tolerance=1.0e-12):
     return species_json
 
 
-def parse_kpp_arrhenius(kpp_str):
+def parse_kpp_arrhenius(kpp_str, to_si_units=False):
     """
     Parse KPP Arrhenius reaction
 
     Parameters
         (str) kpp_str: Arrhenius reaction string
+        (bool) to_si_units: convert A coefficient
+        from cm^3 molecule-1 s-1 to m^3 mol-1 s-1
```

Comment:
I think this should convert from `(cm^3 molecule-1)^(N-1) s-1` to `(m^3 mol-1)^(N-1) s-1` where `N` is the number of reactants, because we don't restrict Arrhenius reactions to have exactly two reactants